tensorflow/lite/CMakeLists.txt

```diff
@@ -503,6 +503,7 @@ elseif(CMAKE_SYSTEM_NAME MATCHES "iOS")
   list(APPEND TFLITE_PROFILER_SRCS
     ${TFLITE_SOURCE_DIR}/profiling/signpost_profiler.mm
   )
+  set_source_files_properties(${TFLITE_SOURCE_DIR}/profiling/signpost_profiler.mm PROPERTIES COMPILE_FLAGS -fno-objc-arc)
```

Comment:
Hi, does bazel has -fno-objc-arc by default?

Reply:
Yes, Bazel has explicitly no ARC for this single file, see https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/profiling/BUILD#L66

Reply:
Hi sorry I didn't see -fno-objc-arc in the link?